lib/transaction.py

```diff
@@ -616,7 +616,7 @@ def BIP_LI01_sort(self):
         self._inputs.sort(key = lambda i: (i['prevout_hash'], i['prevout_n']))
         self._outputs.sort(key = lambda o: (o[2], self.pay_script(o[1])))
 
-    def serialize_output(self, output):
+    def serialize_output(self, output): 
```

Comment:
If you can, try and not include lines like this in your commits.  Set your editor to strip trailing spaces.  I fixed a bunch of these for you already but I am just pointing them out here.  They create extra noise...